internal/cli/down.go

```diff
@@ -82,6 +82,11 @@ func (c *downCmd) down(ctx context.Context, downDeps DownDeps, args []string) er
 		return errors.Wrap(err, "Parsing manifest YAML")
 	}
 
+	// reverse order for tearing down
+	for i, j := 0, len(entities)-1; i < j; i, j = i+1, j-1 {
```

Comment:
I like the spirit of this idea! 

But I don't think this will do what you expect. 

I don't think the order of the manifests is guaranteed to be sorted to begin with, so reversing that order isn't quite right

I think your better bet is reverse-sort all the entities with this function: https://github.com/tilt-dev/tilt/blob/master/internal/k8s/entity.go#L73

Reply:
That was just the pointer I needed! I updated the PR, it looks much cleaner now imo and should also lead to more stable results. I verified it again with both scenarios as described in the original post.